src/Model.php

```diff
@@ -1340,6 +1364,21 @@ public function reload()
         return $this;
     }
 
+    /**
+     * Try to reload model by taking its current ID, otherwise return null.
+     *
+     * @return $this
+     */
+    public function tryReload()
```

Comment:
What are the usecases? If a record is expected to be possibly deleted, you can always check with `tryLoad` or handle `reload` exception.

Reply:
The use case is as follows - consider a Crud which displays unread messages with `addWeakCondition('unread', true)` . You click on edit, and change the unread flag to false. You click save. 
The crud then runs a normal ->save function. Since save allows that - in case a field is changed that moves it outside of a weak condition, the save command should only try to reload. If it is successful, the crud will be updated, if it is null, then the entity left the scope, and thus the same animation as in a delete function will be run.

Reply:
This is legit usecase 👍